brainstormforce/astra
# AST-3477 - Padding issue for above header responsive devices

### Description

- Padding issue for above header responsive devices
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3477

### Screenshots

https://capture.dropbox.com/TFUx06B72aaE3gHB

### Types of changes




Bug fix

### How has this been tested?

- Check Tablet and Mobile values for Above Header after Applying Customizer Padding.
- Check Tablet and Mobile values for Above Header in Customizer Preview as well.
- Check Desktop spacing for Above Header as well.
- Check other header types as well.

### Checklist:
